src/pages/Admin/Charity/Dashboard/Dashboard.tsx

```diff
@@ -12,7 +12,7 @@ export default function Dashboard() {
 
   return (
     <div className="@container w-full max-w-4xl grid content-start mt-6">
-      <Seo title="Charity Dashboard" />
+      <Seo title="Nonprofit Dashboard" />
```

Comment:
Do we need to update "Endowment Designation"' Selector's `"Charity"` value as well?
See:
- https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/master/src/types/aws/ap/index.ts#L30
- https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/master/src/pages/Admin/Charity/EditProfile/Form.tsx#L26
- https://github.com/AngelProtocolFinance/angelprotocol-web-app/blob/master/src/pages/Marketplace/Filter/Designations.tsx#L7

Reply:
> Do we need to update "Endowment Designation"' Selector's "Charity" value as well?
I guess `charity` is one of nonprofit designations, so didn't touch that one